FirebaseDynamicLinks/Sources/FIRDynamicLinks.m

```diff
@@ -440,7 +440,11 @@ - (nullable FIRDynamicLink *)dynamicLinkFromCustomSchemeURL:(NSURL *)url {
       }
     }
   }
-  mainQueueCompletion(nil, nil);
+  mainQueueCompletion(
+      nil, [NSError errorWithDomain:@""
```

Comment:
Domain shouldn't be empty - looks like we use `com.firebase.dynamicLinks` for generic errors in `FIRDynamicLinkNetworking.m`, we should use that again here. Or, if you think this is related to durable deep links, you could use `kFirebaseDurableDeepLinkErrorDomain` as its used elsewhere in this file.

Reply:
Done..Using "com.firebase.dynamicLinks" now.